board/aspeed/evb_ast2700/evb_ast2700.c

```diff
@@ -233,6 +239,20 @@ void aspeed_errata_app_note_6_pcie_ep()
        printf("applying Aspeed app note 6 pcie ep ... Done\n");
 }
 
+void update_por_env(void)
```

Comment:
OpenBMC design document https://github.com/openbmc/docs/blob/master/designs/bmc-reboot-cause-update.md provides the current way of BMC reset handling. In this document "Each BMC vendor needs to record the results of the retrieved reboot cause to the specified path." . and PSM consumes this . Can you look the driver based approch helps to meet AMD requirement?

Reply:
The proposal mentions 
> "Each BMC vendor must provide a driver to retrieve the BMC reboot cause and record the result at the specified location."

I guess we can merge u-boot approach for now and later migrate to when Aspeed provides a driver.